packages/shared/compiled/style-loader/index.js

```diff
@@ -0,0 +1 @@
rom ${n};`:`var styleTagTransformFn = require(${n});`}if(i==="module-path"){const o=stringifyRequest(e,`${n.styleTagTransform}`);e.addBuildDependency(n.styleTagTransform);return t?`import styleTagTransformFn from ${o};`:`var styleTagTransformFn = require(${o});`}return""}function getStyleTagTransformFn(t,e){return e?"":typeof t.styleTagTransform==="function"?`options.styleTagTransform = ${t.styleTagTransform.toString()}`:`options.styleTagTransform = styleTagTransformFn`}function getExportStyleCode(t,e,n){const o=stringifyRequest(e,`!!${n}`);return t?`export * from ${o};\n       export default content && content.locals ? content.locals : undefined;`:"module.exports = content && content.locals || {};"}function getExportLazyStyleCode(t,e,n){const o=stringifyRequest(e,`!!${n}`);return t?`export * from ${o};\n       export default exported;`:"module.exports = exported;"}function getSetAttributesCode(t,e,n){let r;if(typeof n.attributes!=="undefined"){r=n.attributes.nonce!=="undefined"?stringifyRequest(e,`!${o.default.join(__dirname,"runtime/setAttributesWithAttributesAndNonce.js")}`):stringifyRequest(e,`!${o.default.join(__dirname,"runtime/setAttributesWithAttributes.js")}`)}else{r=stringifyRequest(e,`!${o.default.join(__dirname,"runtime/setAttributesWithoutAttributes.js")}`)}return t?`import setAttributes from ${r};`:`var setAttributes = require(${r});`}},17:t=>{t.exports=require("path")},80:t=>{t.exports=JSON.parse('{"title":"Style Loader options","type":"object","properties":{"injectType":{"description":"Allows to setup how styles will be injected into DOM.","link":"https://github.com/webpack-contrib/style-loader#injecttype","enum":["styleTag","singletonStyleTag","autoStyleTag","lazyStyleTag","lazySingletonStyleTag","lazyAutoStyleTag","linkTag"]},"attributes":{"description":"Adds custom attributes to tag.","link":"https://github.com/webpack-contrib/style-loader#attributes","type":"object"},"insert":{"description":"Inserts `<style>`/`<link>` at the given position.","link":"https://github.com/webpack-contrib/style-loader#insert","anyOf":[{"type":"string"},{"instanceof":"Function"}]},"base":{"description":"Sets module ID base for DLLPlugin.","link":"https://github.com/webpack-contrib/style-loader#base","type":"number"},"esModule":{"description":"Use the ES modules syntax.","link":"https://github.com/webpack-contrib/css-loader#esmodule","type":"boolean"},"styleTagTransform":{"description":"Transform tag and css when insert \'style\' tag into the DOM","link":"https://github.com/webpack-contrib/style-loader#styleTagTransform","anyOf":[{"type":"string"},{"instanceof":"Function"}]}},"additionalProperties":false}')}};var e={};function __nccwpck_require__(n){var o=e[n];if(o!==undefined){return o.exports}var r=e[n]={exports:{}};var i=true;try{t[n](r,r.exports,__nccwpck_require__);i=false}finally{if(i)delete e[n]}return r.exports}if(typeof __nccwpck_require__!=="undefined")__nccwpck_require__.ab=__dirname+"/";var n=__nccwpck_require__(529);module.exports=n})();
```

Comment:
Some runtime modules are missing?

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/23891f53-ab5f-4f47-b2a3-e79ad12dd9bf)

Reply:
It seems we need to copy those runtime modules to compiled folder 🤔